.travis.yml

```diff
@@ -1,5 +1,6 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.4.0
+  - 2.1.5
+  - 2.4.1
```

Comment:
I specified two versions here because
- I want this to work well with the newest version,
- I want Jarvis to be able to use this, and
- the tests run really quickly (0.01 sec on my laptop) so there's no reason not to.

I did actually run into a feature that was present in 2.4.1 and not 2.1.5, so it has been useful to have both.